src/vsphere_cpi/lib/cloud/vsphere/vcenter_client.rb

```diff
@@ -458,6 +458,30 @@ def find_child_by_name(mob, child_path)
       find_child_by_name(mob.child_entity.find {|c| c.name == child_entity_name }, child_path)
     end
 
+    def clean_up_vm_groups(cluster, vm_group_names)
```

Comment:
Before merging should refactor this so that these methods are not duplicated in vm_group and here.